.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 <!--
 Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the docs folder covering Actions (actions.md and actions/*) and Plugins (available-plugins.md) are auto-generated.
```

Comment:
This brought up to my attention that we've already moved all of them to `docs/Generated` directory. So:

```suggestion
Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the `/docs/generated` folder are auto-generated.
```